### PR TITLE
Dev add superscript markdown

### PR DESCRIPTION
**Pull request by Zullybur**
#38

### CODE REVIEW

**Comment by forkata** [comment]
:heart:

**Comment by adammathys** [comment]
Superscript should maintain the uppercase styles. (At least, if we want it to match the calendar it should.)

**Comment by Zullybur** [comment]
Adjusted per @adammathys feedback to use uppercase style instead of lowercase.
2016-09-21-133320_618x419_scrot

File: app/assets/stylesheets/imperilment.scss [hunk 1]
```diff
@@ -244,3 +244,9 @@ i {
   padding-bottom: 15px;
   line-height: 20px;
 }
+
+sup {
+  top: 0em;
```
Comment (adammathys) [inline app/assets/stylesheets/imperilment.scss]: > If a value is zero, do not provide a unit.

https://github.com/StemboltHQ/information/blob/master/sass_styleguide.markdown

**Pull request merged**